README.md

```diff
@@ -648,3 +676,12 @@ To do every time you changed code and want to test it:
   - With Stack: `stack build haskell-language-server:exe:haskell-language-server`
 - Restart HLS
   - With VS Code: `Haskell: Restart Haskell LSP Server`
+
+### Adding support for a new editor
+
+Adding support for new editors is fairly easy if the editor already has good support for generic LSP-based extensions.
+In that case, there will likely be an editor-specific support system for this (like `lsp-mode` for Emacs).
+This will typically include instructions for creating a new language server client.
```

Comment:
I don't think this is true. You're not creating a new lsp client - that would mean the entire idea lsp is based on is obsolete. 

And some editors (e.g. (n)vim, kakoune, sublime (as far as I can tell)) directly use their lsp clients without Haskell specific "wrappers" - you only need to do configuration.

Reply:
Okay, I'll make this less prescriptive.

Reply:
Cool, thanks!